algoliasearch-apache/src/main/java/com/algolia/search/apache/ApacheAnalyticsClient.java

```diff
@@ -0,0 +1,38 @@
+package com.algolia.search.apache;
+
+import com.algolia.search.AnalyticsClient;
+import com.algolia.search.AnalyticsConfig;
+import javax.annotation.Nonnull;
+
+/**
+ * Algolia's REST analytics client that wraps an instance of the transporter. which wraps the HTTP
+ * Client This client allows to build typed requests and read typed responses. Requests are made
+ * under the Algolia's retry-strategy. This client is intended to be reused and it's thread-safe.
+ *
+ * @see <a href="https://www.algolia.com/doc/rest-api/analytics/">Algolia.com</a>
+ */
+@SuppressWarnings("WeakerAccess")
+public class ApacheAnalyticsClient {
```

Comment:
I suggest to drop the "Apache" prefix: the code is already in the com.algolia.search.**apache** package.

Reply:
I don't have a strong opinion on it. Do you have example of other libraries in the Java world doing something similar? I couldn't fine one.

Reply:
Actually we don't see it because that is the default construction. Our own classes are for example "com.algolia.search.AccountClient", and not "com.algolia.search.AlgoliaSearchAccountClient"

It's the very purpose of _namespaces_: putting things in the same named set :P

Reply:
Why not having all the `Apache-` classes in `com.algolia.search` then it's very clear for the documentation/customers how to instantiate a client?

I find this (in doc) clearer:
```java
SearchClient client = ApacheSearchClient.create(...);
```

than: 
```java
SearchClient client = com.algolia.search.apache.SearchClient.create(....);
```

ping @aseure

Reply:
Or let's remane `ApacheSearchClient` by `DefaultSearchClient`

Reply:
This sounds like a good approach to me 👍